app/jinja_filters.py

```diff
@@ -279,7 +279,16 @@ class OtherConfig:
     def __init__(self, detail_answer):
         self.id = detail_answer.id
         self.name = detail_answer.name
-        self.value = detail_answer.data or ''
+
+        if detail_answer.data or detail_answer.data == 0:
+            self.value = detail_answer.data
+        elif detail_answer.raw_data and (
+            detail_answer.raw_data[0] or detail_answer.raw_data[0] == 0
```

Comment:
Don't believe you need the detail_answer.raw_data[0] == 0 check as '0' would be valid and therefore caught in the first if

Reply:
It was needed because even though 0 is valid, it would be falsy. But it doesn't matter now because I've removed this to address Andrews comment